README.md

```diff
@@ -26,10 +26,18 @@ See ``config.yaml.dist`` for a configuration example.
 
 ## Run
 
+Native:
+
 ```
 $ ./mqtt_blackbox_exporter -config.file config.yaml
 ```
 
+Using Docker:
+
+```
+docker run --rm -it -p 9142:9142 -v ${PWD}/:/data/ inovex/mqtt_blackbox_exporter:latest -config.file /data/config.yaml
```

Comment:
The exporter itself currently uses port 9142 as default, which I suggest to change. That part is unrelated this PR.  

But taking this port and using it as default of the Docker image also is part of this PR. How about we get our "own" port before we merge PR?